src/internal_macros.rs

```diff
@@ -227,7 +227,7 @@ macro_rules! display_from_debug {
 macro_rules! hex_script (($s:expr) => (::blockdata::script::Script::from(::hex::decode($s).unwrap())));
 
 #[cfg(test)]
-macro_rules! hex_hash (($h:ident, $s:expr) => ($h::from_slice(&::hex::decode($s).unwrap()).unwrap()));
+macro_rules! from_hex (($s:expr) => (::hashes::hex::FromHex::from_hex($s).unwrap()));
```

Comment:
It might be worth naming this function `hash_from_hex` or somehow similarly to indicate that it returns Hash structure, not just converts random hex string into Vec or something

Reply:
It can do `Vec<u8>` as well. Anything that has `FromHex` implemented can be parsed with this.

You can do `let bytes: Vec<u8> = from_hex!("aabb");`.